kytos/templates/kytos.conf.template

```diff
@@ -70,6 +70,13 @@ vlan_pool = {}
 # The jwt_secret parameter is responsible for signing JSON Web Tokens.
 jwt_secret = {{ jwt_secret }}
 
+# Maximum number of thread workers in a thread pool, if it's set as empty dict {} no thread pools will be used.
+# The following pools are available by default to be used in the listen_to decorator:
+# - sb: it's used automatically by kytos/of_core.* events, it's meant for southbound related messages
+# - app: it's meant for general NApps event, it's default pool if no other one has been specified
+# - db: it can be used by for higher priority db related tasks (need to be parametrized on decorator)
+thread_pool_max_workers = {"sb": 256, "db": 256, "app": 512}
```

Comment:
I've bumped the default `app` to 512 instead of 256 to be more accommodating for NApps and other medium sized topology based on the original issue that Italo has reported, in a scenario with 20 req / secs, not much different was observed in terms of consumed memory (comparing 512 - the first chart vs 256 the second chart), this won't allocate memory upfront, it only will have a greater pool and higher queue size. 

![plot_t512_20req_s](https://user-images.githubusercontent.com/1010796/166829051-4171f8cb-a9c8-464a-bb00-2d098b108342.png)

![plot_t256_20req_s](https://user-images.githubusercontent.com/1010796/166829078-fb05cb77-6adb-403c-85ad-10ac02377b19.png)